tasks.rb

```diff
@@ -2092,6 +2092,9 @@ def apply_hpxml_modification_sample_files(hpxml_path, hpxml)
     elsif ['base-dhw-tank-gas-fhr.xml'].include? hpxml_file
       hpxml_bldg.water_heating_systems[0].first_hour_rating = 56.0
       hpxml_bldg.water_heating_systems[0].usage_bin = nil
+    elsif ['base-dhw-tank-heat-pump-containment-volume.xml'].include? hpxml_file
```

Comment:
Can we rename the sample file to `base-dhw-tank-heat-pump-confined-space.xml`? Every water heater has a surrounding volume, but the point here is that the surrounding volume in this test file is small (i.e., it's a confined space).